### PR TITLE
Disable validation via a global property

### PR DESCRIPTION
**Pull request by rcrichton**
See https://issues.openmrs.org/browse/TRUNK-4626

### CODE REVIEW

File: webapp/pom.xml [hunk 1]
```diff
@@ -94,7 +94,7 @@
          <plugin>
             <groupId>org.mortbay.jetty</groupId>
             <artifactId>maven-jetty-plugin</artifactId>
-            <version>6.1.10</version>
+            <version>6.1.26</version>
```
Comment (kristopherschmidt) [inline webapp/pom.xml]: Is there a reason to go to another 6.* version of Jetty, is it possible to go to a recent stable version of Jetty 8? One reason I ask is that core is now building against Servlet 3.0 and there have been discussions about utilizing that functionality, so an upgrade of Jetty to 8 would be ideal if it works out.

Comment (sunbiz) [inline webapp/pom.xml]: I agree... with the change in master to servlet 3, we might as well go to jetty 8. But may be we need a separate ticket? I made the below change to the webapp/pom.xml and everything seems to work with master as expected. Not sure if it solves Ryan's classloader problem, which this ticket is about.
<groupId>org.mortbay.jetty</groupId>
<artifactId>jetty-maven-plugin</artifactId>
<version>8.1.16.v20140903</version>

Comment (djazayeri) [inline webapp/pom.xml]: I vote for merging this with a 6.x version of Jetty. I presume Ryan will want to backport this to a released maintenance line.

Then in another ticket (that won't be backported) we can go to Jetty 8.x.

Comment (rcrichton) [inline webapp/pom.xml]: Yeah, I didn't want to be too disruptive but the latest version of jetty worked for me to. I agree with @djazayeri another issue would be the best place to do this.

Comment (kristopherschmidt) [inline webapp/pom.xml]: I opened TRUNK-4638 for upgrading to Jetty 8. (I'm not claiming the ticket
though, so anybody that wants to take this feel free)

**Comment by dkayiwa** [comment]
@rcrichton this pull request has merge conflicts. Do you think you can pull the latest changes from upstream and try again?

**Comment by dkayiwa** [comment]
@rcrichton remind me to reopen this when you resume working on it.

**Comment by rcrichton** [comment]
@dkayiwa I've merged in the latest from master. This should be good to go now.

**Comment by dkayiwa** [comment]
@rcrichton can you also squash these commits into one?

**Comment by rcrichton** [comment]
Hey @dkayiwa I've squashed the commits and merged with latest from upstream. I see this PR hasn't updated with the latest changes, I'm not sure if that is because it is closed. Let me know if I need to create another PR or if it works if you open it again.

**Comment by dkayiwa** [comment]
@rcrichton unfortunately i cannot reopen this pull request because the branch was recreated or force pushed. Do you mind creating a new pull request?

**Comment by rcrichton** [comment]
@dkayiwa ok, I've created a new PR https://github.com/openmrs/openmrs-core/pull/1494. It must have been because of the force push to squash the commits.

**Pull request closed without merging**